sonic-slave-stretch/Dockerfile.j2

```diff
@@ -273,8 +273,8 @@ RUN apt-get update && apt-get install -y \
         librrd-dev \
         rrdtool \
 # For smartmontools 6.6-1
-        automake1.11 \
-        libselinux1-dev \
+#        automake1.11 \
+#        libselinux1-dev \
```

Comment:
Reason these are commented and not removed as in the other slave containers?

Reply:
should be removed